CHANGELOG.md

```diff
@@ -7,7 +7,7 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
-## Unreleased
+## 16.8.0
 * Add date input component based on GOV.UK Frontend (PR #792)
```

Comment:
Nitpick: we seem to have lost the newline after the release heading for this and the last release.

Reply:
Good catch, must have got lost in rebase. Updated.